<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="prikab" name="SEGMENTS RESTORE TESTCASES" tests="1" failures="0" timestamp="9 Nov 2016 02:17:37 GMT" time="0.014" errors="0">
  <testcase name="@BeforeMethod beforeTest" time="0.0" classname="configuration.SegmentsRestore_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.beforeTest(TestBase.java:87)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeTest -->
  <testcase name="@AfterMethod AfterLogin" time="0.0" classname="configuration.SegmentsRestore_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.AfterLogin(TestBase.java:105)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod AfterLogin -->
  <testcase name="validate_segments_page" time="0.0" classname="configuration.SegmentsRestore_testcases">
    <skipped/>
  </testcase> <!-- validate_segments_page -->
</testsuite> <!-- SEGMENTS RESTORE TESTCASES -->
